src/settings.py

```diff
@@ -120,7 +120,7 @@ class GlobalSettings:
     grammar_search_pred_complexity_weight = 1
     grammar_search_max_predicates = 50
     grammar_search_predicate_cost_upper_bound = 6
-    grammar_search_score_function = "hadd_lookahead_depth1"
+    grammar_search_score_function = "hff_lookahead_depth1"
```

Comment:
i'm still nervous about using hff with lookahead, but I guess we're going to immediately investigate this so it's fine

Reply:
yup, exactly. it's the only one that's doing the right thing in painting